include/groonga/groonga.h

```diff
@@ -1131,31 +1131,40 @@ grn_obj_close(grn_ctx *ctx, grn_obj *obj);
  */
 GRN_API grn_rc
 grn_obj_reinit(grn_ctx *ctx, grn_obj *obj, grn_id domain, uint8_t flags);
-/* On non reference count mode (default):
- * This closes the following objects immediately:
+/**
+ * \brief Unlink an object.
+ *
+ *        This function releases the specified object (`obj`) from memory.
+ *        If the object has dependent objects, they are also recursively
+ *        freed. The behavior of this function varies depending on whether
+ *        reference counting is enabled.
  *
- *   * Acceessor
- *   * Bulk
- *   * DB
- *   * Temporary column
- *   * Temporary table
+ * Non reference count mode (default: GRN_ENABLE_REFERENCE_COUNT=no)
```

Comment:
```suggestion
 * Non reference count mode (default: `GRN_ENABLE_REFERENCE_COUNT=no`):
```

それはそうといきなり`GRN_ENABLE_REFERENCE_COUNT=no`と書かれてもなんのことかわからなくないっすか？